release-engineering/role-handbooks/branch-manager.md

```diff
@@ -608,7 +608,7 @@ Here's an [example PR](https://github.com/kubernetes/test-infra/pull/15015).
 6. [Announce in #sig-release and #release-management](https://kubernetes.slack.com/archives/C2C40FMNF/p1565746110248300?thread_ts=1565701466.241200&cid=C2C40FMNF) that this work has been completed
 
 [sig-release-x.y-blocking]: https://testgrid.k8s.io/sig-release-1.17-blocking
-[`branchff`]: https://git.k8s.io/release/branchff
+[`krel ff`]: https://git.k8s.io/release/cmd/krel/cmd/ff.go
```

Comment:
Don't use this as the link. It will encourage misuse.

Reply:
Changed the link to https://git.k8s.io/release, we probably want to add a dedicated README.md for krel or at least mention it in the main README.md there.